Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
+
+'Neos.Neos:Document':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
same here, doesn't it defeat the whole purpose of the `.Document` prototype which is to enable auto assigned properties in a usable way for document node types?

Reply:
This is not needed any more since with 4.0 this is the default in the core ;-) for Node and Document